### PR TITLE
REVSDL-2137: fix sending OnHMIStatus to mobile if SDL receives OnDevi…

### PR DESCRIPTION
**Pull request by bonewell**
…ceLocationChanged

### CODE REVIEW

File: src/components/application_manager/src/policies/policy_handler.cc [hunk 1]
```diff
@@ -1626,5 +1613,44 @@ void PolicyHandler::OnRemoteAppPermissionsChanged(const std::string& device_id,
...
+  if (mobile_apis::HMILevel::INVALID_ENUM == level) {
+    LOG4CXX_WARN(logger_, "Couldn't convert default hmi level "
+                 << hmi_level << " to enum.");
+    return;
+  }
+  mobile_apis::DeviceRank::eType rank = MessageHelper::StringToDeviceRank(device_rank);
+  if (rank == mobile_apis::DeviceRank::INVALID_ENUM) {
+    LOG4CXX_WARN(logger_, "Couldn't convert device rank "
+                 << device_rank << " to enum.");
+  }
```
Comment (VladSemenyuk) [inline src/components/application_manager/src/policies/policy_handler.cc]: "return" doesnt need?

Comment (VladSemenyuk) [inline src/components/application_manager/src/policies/policy_handler.cc]: exactly need))

Comment (bonewell) [inline src/components/application_manager/src/policies/policy_handler.cc]: Fixed.

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**